scripts/download_pages.py

```diff
@@ -17,28 +17,26 @@
 not be a big problem. Too bad we have to open and close file handles all the
 time.
 """
-
+import queue
 from argparse import ArgumentParser
-from concurrent.futures import ThreadPoolExecutor
+import boto3
 import gzip
 import hashlib
-from itertools import groupby
 import logging
-from operator import itemgetter
 import os
 from pathlib import Path
-from queue import Empty, Full, Queue
-import signal
 import subprocess
 from tempfile import gettempdir
 import threading
 import time
-from typing import TextIO
+from typing import Any, TextIO
 import zlib
 
 from cc_corpus.download import download_warc_ranges, DownloadError
 from cc_corpus.utils import notempty, num_digits, openall, otqdm
 
+BUCKET_NAME = 'commoncrawl'
```

Comment:
We don't really need this, do we?

Reply:
Right, in the end it was not used there.